sibyl/db/preprocessing.py

```diff
@@ -376,7 +376,7 @@ def _process_fp(fn):
                     dataset_fp,
                     target,
                     set_doc,
-                    name=explainer_file[:-4],  # remove .pkl
+                    model_id=explainer_file[:-4],  # remove .pkl
                     pickle_model_fp=_process_fp(cfg.get("pickle_model_fn")),
```

Comment:
Don't we need to use different pickle models for each model in the directory as well?

Reply:
Right now we allow users to either give a list of models or list of explainers. The model's aren't used at all if the explainers are passed in. I've removed the model arg here to avoid confusion